home/models.py

```diff
@@ -532,6 +551,20 @@ def whatsapp_template_body(self):
     def whatsapp_template_image(self):
         return self.whatsapp_body.raw_data[0]["value"]["image"]
 
+    @property
+    def whatsapp_template_example_values(self):
+        # TO CHECK IN REVIEW - I know there must be a cleaner way of doing this, but I can't seem to come up with it
```

Comment:
Maybe a:
```
example_values = self.whatsapp_body.raw_data[0]["value"].get("example_values", [])
return [v["value"]["example_values"] for v in example_values]
````

Reply:
Rad, thanks.  I modified it slightly as the structure changed when I removed the StructBlocks